src/Controls/tests/TestCases.HostApp/Issues/Issue30052.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 30052, "Right-To-Left (RTL) alignment is not applied to Editor placeholder", PlatformAffected.iOS)]
+public class Issue30052 : TestContentPage
+{
+	protected override void Init()
+	{
+		Content = new VerticalStackLayout
+		{
+			FlowDirection = FlowDirection.RightToLeft,
+			Children =
```

Comment:
Could the test also verify changes at runtime? like change the FlowDirection tapping a Button.